### PR TITLE
add crnk.io Java implementation

### PR DESCRIPTION
**Pull request by remmeier**
www.crnk.io is a new Java framework for JSON API.

### CODE REVIEW

**Comment by remmeier** [comment]
FYI: let me know if there is any help needed to merge the various implementation PRs. As this page seems a popular starting point for many developers, it would be great if this and the others project get listed.

File: implementations/index.md [hunk 1]
```diff
@@ -188,6 +190,10 @@ includes related resources.
 
 ### <a href="#server-libraries-java" id="server-libraries-java" class="headerlink"></a> Java
 
+
+* [crnk.io](https://www.crnk.io) is a JSON API framework for clients and servers. On the server-side it comes with a rich set
+  of integrations (Servlet, JAX-RS, Spring, JPA, Bean Validation, Zipkin and more), bulk updates with JSON Patch, client stub 
+  generation for TypeScript and a module API for third-party contributions.
```
Comment (dgeb) [inline implementations/index.md]: Could you please move this to the bottom of the list? We try to keep the lists chronological.

Comment (remmeier) [reply implementations/index.md]: done, makes sense

**Comment by steveklabnik** [comment]
Thanks!

**Pull request merged**